modules/app_modules/coar_notify.py

```diff
@@ -183,14 +183,14 @@ def _recommendation_as_jsonld(self, recommendation):
     def _article_as_jsonld(self, article):
         return {
             "id": article.doi,
```

Comment:
article.doi should also be replaced by article_cite_as(article) to include version number